using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
 Deleting your mobile property in Experience Platform Launch does not automatically delete your Campaign Standard mobile app.
 {% endhint %}
 
+### Handling deeplinks included in Campaign In-App messages
+
+A deeplink can be added to in-app messages delivered from Adobe Campaign. The link can be a website URL such as https://www.adobe.com or an app link which can be used to direct the user to a specific area of your app.
```

Comment:
- In the 1st sentence, please add "that" between "in-app messages" and "delivered".
-In the 2nd sentence, please add a comma between "link" and "which".